examples/kubernetes/wca-scheduler/README.rst

```diff
@@ -44,3 +44,77 @@ Push image to repository:
 Check if wca-scheduler pod is running:
 
 ``kubectl apply -k .``
+
+
+TLS connection wca-scheduler with kube-scheduler
+------------------------------------------------
+
+TLS is required to secure a connection between wca-scheduler and kube-scheduler.
+
+Instructions are based on https://kubernetes.io/docs/tasks/tls/managing-tls-in-a-cluster/
+
+Required are the cfssl tools from https://pkg.cfssl.org/ to generate the private key and
```

Comment:
Here I would write step by step instruction instead of directing a user to links. I've just started following the instruction and don't know what to do without jumping around the links you provided.